### DISCUSSION TITLE
Convention sur les noms de fichiers

### DISCUSSION THREAD

**Discussion by tpoisot** [BIO6032/2022-SEIR-reproduction#2] [General]
- noms des dossiers
- extensions
- ...

**Comment by tpoisot** [comment]
Quelques informations pertinentes: https://libguides.princeton.edu/c.php?g=102546&p=930626

**Comment by Marine1719** [comment]
Est-ce qu'on part sur des noms de dossiers avec la date, "RPOJECT" et les sections ? 
- Par exemple 20220322_Project_Objectifs - 20220322_Project_model_description - 20220322_Project_equations, etc

**Reply by blanchette31** [reply]
Pour éviter que les noms de fichiers deviennent trop longs, on pourrait simplement dire "20220322_model_description" et couper "Project". Ça permet de couper aussi sur la redondance également. Je crois que les noms des fichiers devraient être courts mais précis. Qu'en pensez-vous?

**Reply by aushay** [reply]
Je préfère aussi sans le Project. L'utilisation des majuscules me paraît intéressante.

**Comment by Marine1719** [comment]
- [x] Fichier code
- [x] Fichier figures avec figure 1, 2 et 3 
- [x] Fichier data
